pkg/sql/parser/sql.y

```diff
@@ -1867,6 +1873,65 @@ opt_validate_behavior:
     $$.val = tree.ValidationDefault
   }
 
+// %Help: ALTER TYPE - change the definition of a type.
+// %Category: DDL
+// %Text: ALTER TYPE <typename> <command>
+//
+// Commands:
+//   ALTER TYPE ... ADD VALUE <value> [ { BEFORE | AFTER } <value> ]
```

Comment:
i'm looking at https://www.postgresql.org/docs/current/sql-altertype.html and we seem to be missing a few. should we add those in?

Reply:
I left them out because I don't think we have any upcoming plans to implement them (I can if you think I should though)

Reply:
worth adding for metricss' sake, at the very least..

Reply:
done

wew that was a bunch of extra yaccing around